server/weaverbird/backends/sql_translator/steps/utils/query_transformation.py

```diff
@@ -386,6 +386,12 @@ def sanitize_input(value: str) -> str:
     return value.replace('"', '\\"').replace("'", "\\'")
 
 
+def sanitize_column_name(col: str) -> str:
```

Comment:
Maybe we should handle also:  
- spaces ?
- single/double quotes ?
- not alphanumerics characters

Reply:
Thanks for this remark. 
* Whitespace is covered 
* It might be dangerous to replace quotes as it is valid to have quotes in column names : https://docs.snowflake.com/en/sql-reference/identifiers-syntax.html
* Alphanumeric caracters are valid if not at first position of the column name (but I improve the replacement)

I add a unit test :+1: